linux/ibus-keyman/tests/run-tests.sh

```diff
@@ -30,34 +30,24 @@ function cleanup() {
   fi
 }
 
+while (( $# )); do
+  case $1 in
+    -k) ARG_K=$1 ;;
+    --tap) ARG_TAP=$1 ;;
+    --surrounding-text) ARG_SURROUNDING_TEXT=$1 ;;
+    --no-surrounding-text) ARG_NO_SURROUNDING_TEXT=$1 ;;
+    --) shift && break ;;
+    *) echo "Error: Unexpected argument \"$1\". Exiting." ; exit 4 ;;
+  esac
+  shift || (echo "Error: The last argument is missing a value. Exiting."; false) || exit 5
+done
```

Comment:
It'd be good to document these flags with `--help|-h|-?` 😁

Reply:
Done